hypertrace-graphql-service/src/main/java/org/hypertrace/graphql/service/DefaultGraphQlServiceConfig.java

```diff
@@ -22,16 +24,19 @@
 
   private static final String ATTRIBUTE_SERVICE_HOST_PROPERTY = "attribute.service.host";
   private static final String ATTRIBUTE_SERVICE_PORT_PROPERTY = "attribute.service.port";
+  private static final String ATTRIBUTE_SERVICE_PORT_TIMEOUT = "attribute.service.timeout";
```

Comment:
why is this ATTRIBUTE_SERVICE_PORT_TIMEOUT instead of ATTRIBUTE_SERVICE_CLIENT_TIMEOUT?

Reply:
Typo, addressed.